tests/integration/reportinglib/1.check.in

```diff
@@ -4,18 +4,45 @@
 
 OK=0
 FAILED=1
+sonata_reports=@ENABLE_SONATA_REPORTS@
+reportinglib=@ENABLE_REPORTINGLIB@
 
-if [ -f test_1.bbp ] 
+if [ "$reportinglib" = "ON" ]
 then
-   #checking something with @reportinglib_BIN_DIR@/dump test_1.bbp
-   # for ascii files that will be using grep for instance.
-   nb_recorded=$(@reportinglib_somaDump@ test_1.bbp 1| wc -l)
-   if [ $nb_recorded -eq 100 ]; then
-     exit $OK
-   else
+  if [ -f test_1.bbp ]
+  then
+     #checking something with @reportinglib_BIN_DIR@/dump test_1.bbp
+     # for ascii files that will be using grep for instance.
+     nb_recorded=$(@reportinglib_LIB_DIR@/../usr/local/bin/somaDump test_1.bbp 1| wc -l)
```

Comment:
You should make sure that `somaDump` executable exists before calling it. Actually, it should be done by CMake and have a `@SomaDump_EXECUTABLE@` variable